ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -392,107 +343,55 @@ private void createBanner() {
       return;
     }
 
-    cleanUp();
+    VungleManager.getInstance().cleanUpActiveBannerAd(mPlacementId);
     RelativeLayout.LayoutParams adParams =
         new RelativeLayout.LayoutParams(
             RelativeLayout.LayoutParams.WRAP_CONTENT, RelativeLayout.LayoutParams.WRAP_CONTENT);
     adParams.addRule(RelativeLayout.CENTER_HORIZONTAL, RelativeLayout.TRUE);
     adParams.addRule(RelativeLayout.CENTER_VERTICAL, RelativeLayout.TRUE);
-    VungleListener listener = mVungleListener.get();
+    MediationBannerListener listener = bannerListener.get();
```

Comment:
minor: adapter.get() can be stored to local variable here

Reply:
done.